Makefile

```
@@ -3,7 +3,7 @@ BUNKER_MINTER_PACKAGE=teritori-bunker-minter
 GO?=go
 GOFMT?=$(shell $(GO) env GOROOT)/bin/gofmt
 
-COSMWASM_CONTRACTS_DIR=cosmwasm-contracts
+COSMWASM_CONTRACTS_DIR=rust
```

```suggestion
COSMWASM_CONTRACTS_DIR=rust/cw-contracts
```

so you don't need the filter bellow
`if [[ $${indir} == "rust/apipb" ]]; then continue; fi ; \`